docs/content/preview/architecture/docdb/persistence.md

```diff
@@ -102,7 +102,56 @@ The non-primary key columns correspond to subdocuments within the document. The
 
 We use a binary-comparable encoding to translate the value for each YCQL type to strings that go to the KV-Store.
 
-### Data expiration in YCQL
+## Packed Row format (Beta)
+
+A row corresponding to the user table is stored as multiple key value pairs in the DocDB. For
+example, A row with one primary key and n non-key columns - K (primary key)  |  C1  | C2  | ………  |
+Cn, would be stored as n key value pairs - <K, C1> <K, C2> .... <K, Cn>. 
+
+With packed row format, it would be stored as a single key value pair <K, packed {C1, C2...Cn}>.
+
+While UDTs can be used to achieve the packed row format at the application level, native support for
+packed row format has the following benefits:
+* Lower storage footprint.
+* Efficient INSERTs, especially when a table has large number of columns.
+* Faster bulk ingestion.
+* Faster multi-column reads, as the reads now need to fetch fewer key value pairs.
+* UDTs require application rewrite, and thus are not necessarily an option for everyone, like latency sensitive update workloads.
```

Comment:
I almost feel like an application re-write could be it's standalone point
while latency sensitive updates could be a separate point -- but we'd need to call out how updates work in general and how they stay the same for packed

Reply:
The 'Design' section talks about the updates. Updates work the same as before and the lookup part of the update might be faster (that point is already captured above. Do you think its worth adding additional references to update perf here?